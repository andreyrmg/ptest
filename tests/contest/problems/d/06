1
895
677 2 83983