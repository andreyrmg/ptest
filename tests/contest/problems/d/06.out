2 122 8
